package/MDAnalysis/analysis/hbonds/hbond_analysis.py

```diff
@@ -1037,17 +1038,12 @@ def _get_timestep():
     @staticmethod
```

Comment:
Can you get rid of the method completely and just call `distances.calc_angle()` in the code?

Reply:
could do, it'd technically be a API break but yeah